cmd/resultserver/main.go

```diff
@@ -123,7 +123,7 @@ func server(c *config) {
 		} else if encoding == "bzip2" {
 			extraExtension = "." + extraExtension
 		}
-		// TODO(jaosorior): Check that content-type is application/xml
+		// TODO(jaosorior): CheckResult that content-type is application/xml
```

Comment:
lol, let's revert back to the original message here.

Reply:
fixed locally